packages/flame/lib/src/game/flame_game.dart

```diff
@@ -78,7 +75,7 @@ class FlameGame extends Component with Game {
   @override
   @mustCallSuper
   void update(double dt) {
-    super.update(dt);
+    super.updateTree(dt, callUpdate: false);
```

Comment:
Instead of having a special parameter `callUpdate`, which is used just in this class, it's cleaner to simply re-define `updateTree()` within this class.

Reply:
That won't work if the game is not the root component, realized that this won't work in that case either since `updateTree` will then be called twice.

Reply:
I think this would work then:
```dart
  @override
  void update(double dt) {
    if (parent == null) {
      children.updateComponentList();
      children.forEach((c) => c.updateTree(dt));
    }
    _cameraWrapper.update(dt);
  }

  @override
  void updateTree(double dt) => update(dt);
```

Reply:
Isn't it nicer to not duplicate the code, in case we would change component.updateTree in the future? Did you see the update? This comment is on an outdated file.